investd/sources/__init__.py

```diff
@@ -15,12 +16,10 @@
 ]
 
 
-def ingest_sources_as_df() -> pd.DataFrame:
-    df_tx = pd.DataFrame(columns=Transaction.__dataclass_fields__.keys())
-    for source_class in sources:
-        source = source_class()
-        df_tx = pd.concat(
-            [df_tx, pd.DataFrame([tx for tx in source.load_transactions()])]
-        )
+def ingest_all() -> pd.DataFrame:
+    df_tx = pd.DataFrame(
+        data=chain.from_iterable(Source().load_transactions() for Source in sources),
+        columns=(field.name for field in fields(Transaction)),
```

Comment:
my IDE shows a type error for columns here (that it is a generator but it shouldn't. For some reason no warning about the "data" key